service/src/main_impl.rs

```diff
@@ -985,17 +1081,17 @@ fn register_collateral(
 		let (fmspc, _tcb_info) = extract_tcb_info_from_raw_dcap_quote(&dcap_quote).unwrap();
 		println!("[>] DCAP setup: register QE collateral");
 		let uxt = enclave.generate_register_quoting_enclave_extrinsic(fmspc).unwrap();
-		send_extrinsic(uxt, api, accountid, is_development_mode);
+		send_integritee_extrinsic(uxt, api, accountid, is_development_mode);
 
 		println!("[>] DCAP setup: register TCB info");
 		let uxt = enclave.generate_register_tcb_info_extrinsic(fmspc).unwrap();
-		send_extrinsic(uxt, api, accountid, is_development_mode);
+		send_integritee_extrinsic(uxt, api, accountid, is_development_mode);
 	}
 }
 
-fn send_extrinsic(
+fn send_integritee_extrinsic(
```

Comment:
no need to be generic over parentchains. we only need this for Integritee Network